Microsoft.Toolkit.Uwp.UI.Controls/AdMediator/UwpAdMediator.xaml

```diff
@@ -0,0 +1,13 @@
+﻿<UserControl x:Class="Microsoft.Toolkit.Uwp.UI.Controls.UwpAdMediator"
+             xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+             xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+             xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls"
+             xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+             xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+             mc:Ignorable="d"
+             d:DesignHeight="300"
+             d:DesignWidth="400">
+    <Grid x:Name="myAdGrid"
```

Comment:
I'm guessing all this should have been in a Generic.xaml instead of relying on user control